modules/bookActions.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable */
```

Comment:
- [ ] Kindly remove this `eslint-disable` command in all of your folders. Eslint checks exist to ensure that you follow best JavaScript practices while coding. It is the industry standard and any company has their own coding norms which you must follow so kindly remove this disable check and fix your files accordingly